### ISSUE TITLE
Unable to get events in IOS same as Android

### ISSUE THREAD

**Issue by sumitHalder786** [mieszko4/react-native-zoom-us#96]
Able to get all the **MeetingEvent** in android.., but in IOS is not the same.., when are you going to address this...?? can you please fix this issue as earliest as possible!!

Thanks in advance!!

**Comment by Alessandro1918** [comment]
Hello, @sumitHalder786 !

I'm currently developing a project with that same package as you. Just a heads-up, without any logs or error messages it will be hard for the people here in the internet to know what's not working for you. What logs / prints can you share of your error?

Also, since you said you got the **MeetingEvent** working on Android, did you manage for your demo app to join in an already started meeting? After you clicked on the "Join Example Meeting" and get the  `LOG  {"event": "success", "status": "MEETING_STATUS_CONNECTING"}` event, what happened?

**Comment by sumeetHalder** [comment]
Thanks for reverting back soon!! much appreciated  **Alessandro1918**!

Hey, i am not getting any error message.., I am just trying to say that when we clicked on "Join Example Meeting", below are the events is getting triggered for Android:
 **MEETING_STATUS_CONNECTING**, 
 **MEETING_STATUS_IN_WAITING_ROOM** ,
 **MEETING_STATUS_RECONNECTING**,
 **MEETING_STATUS_INMEETING**, etc...,
 
 But in IOS, is not the same, i can only able to see events like {"event":"success", "status":undefined} and {"event":"endedBySelf", "status":undefined}..etc..,  these are the couple of events which i am getting in IOS. 
 
 was just expecting the events same as android !!
 
 Thanks!

**Comment by Alessandro1918** [comment]
Weird... I'm having issues with the Android part (your comment saying at least _that_ worked for you is a beacon of hope!), but I'm getting more logs than you.

- Code is pretty much the same as [react-native-zoom-us-test](https://github.com/mieszko4/react-native-zoom-us-test) . I init the sdk with "clientKey", "clientSecret", and for "meetingNumber", I used my Personal Meeting ID
(by the way, there's a typo in the code for App.tsx; var `skdKey `should be named `sdkKey`. It doesn't error because it's named like this thru all the code)

- Again at [react-native-zoom-us-test](https://github.com/mieszko4/react-native-zoom-us-test), I tried to update the package.json dependencies of "react-native-zoom-us", but both versions got me the same error 
Original:
```
"dependencies": {
    ...
    "react-native-zoom-us": "5.8.4"
  },
```
Updated to: 
```
    "react-native-zoom-us": "^5.12.0"
```
- I get this "MEETING_STATUS_DISCONNECTING" right after input my password and hit "ok" to join a meeting. I think "meetingClientIncompatible" may be related to old SDK versions, but not sure yet...


Don't have an iOS device to test your issue, sorry... but if you did something else on your test that I haven't, and think this may cause this error on my Android, that would be great!

Thanks!

![Capturar](https://user-images.githubusercontent.com/54996931/129750628-83f37413-fa7f-4d90-afe7-008d5cdd708a.PNG)

**Comment by sumeetHalder** [comment]
Hi Alessandro1918!

Did you get the chance to check that issue in IOS,?? 

**[Issue: Not getting proper event status as like as android ]**

Will be much appreciate, if you could resolve this problem as earliest as possible.

Thanks!!

**Comment by Alessandro1918** [comment]
Nope, didn't get to work neither on Android or iOS. My project changed directions a little and I'm not on this topic anymore... but good luck on yours!

**Comment by mieszko4** [comment]
I've fixed this event inconcistency in v6.1.1 (https://github.com/mieszko4/react-native-zoom-us/commit/d44363cf75ec0aac737980bfce542e06a80403c8)
